src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs

```diff
@@ -371,6 +390,16 @@ internal override ImmutableArray<Cci.INamespaceTypeDefinition> GetAnonymousTypes
             return StaticCast<Cci.INamespaceTypeDefinition>.From(Compilation.AnonymousTypeManager.GetAllCreatedTemplates());
         }
 
+        internal override ImmutableArray<Cci.INamespaceTypeDefinition> GetMethodGroupConversionCacheFrames()
```

Comment:
Can a frame eve be a nested type? Cci.INamespaceTypeDefinition cannot represent a nested type.